lib/web/apiserver_test.go

```diff
@@ -7993,10 +8000,6 @@ func decodeSessionCookie(t *testing.T, value string) (sessionID string) {
 	return cookie.SessionID
 }
 
-func (r CreateSessionResponse) response() (*CreateSessionResponse, error) {
```

Comment:
I think this is a remain from a bygone time as it doesn't make much sense anymore, so removed.